src/mavedb/routers/scoresets.py

```diff
@@ -174,6 +177,25 @@ async def get_scoreset_counts_csv(
     return StreamingResponse(iter([stream.getvalue()]), media_type="text/csv")
 
 
+@router.get(
+    "/scoresets/{urn}/mappedVariants", status_code=200, response_model=list[mapped_variant.MappedVariant])
```

Comment:
Should be `mapped-variants` to be consistent with other API endpoints right?

Reply:
You're right. I'll change it.